HHS/OPRE-OPS
# OPS-1017 agreement PUT and PATCH

## What changed

adds PUT and PATCH to /agreements/<id>

## Issue

[OPS-1017](https://github.com/HHS/OPRE-OPS/issues/1017)

## How to test

see pytests:
`test_agreements_put_by_id` and `test_agreements_patch_by_id` in
`ops_api/tests/ops/agreement/test_agreement.py`
or execute similar PUT and PATCH requests on `/api/v1/agreements/<id>`
